src/models/sequelize.ts

```diff
@@ -14,5 +14,5 @@ config.logging = config.disable_sql_logging
 	: (query, ms) => logger.trace({ ms }, query);
 
 export const sequelize = config.use_env_variable
-	? new Sequelize.Sequelize(process.env[config.use_env_variable], config)
+	? new Sequelize.Sequelize(process.env[config.use_env_variable] || "DATABASE_URL", config)
```

Comment:
Why fall back to "DATABASE_URL"? To make a potential error message better readable?

Reply:
Whoops, that was a good catch.  the strict checks found that `process.env[config.use_env_variable]` could be undefined.  The way I've done it is wrong though, it's been fixed in another commit.